src/rules/noInferrableTypesRule.ts

```diff
@@ -27,13 +27,23 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 class NoInferrableTypesWalker extends Lint.RuleWalker {
+    public ignoreParameters: boolean = false;
```

Comment:
Define a `const OPTION_IGNORE_PARAMS = "ignore-params";` above the `class Rule` declaration.